smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/serde/json/MemberShapeDecodeGenerator.kt

```diff
@@ -1,318 +0,0 @@
-/*
- * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
- * SPDX-License-Identifier: Apache-2.0.
- */
-package software.amazon.smithy.swift.codegen.integration.serde.json
-
-import software.amazon.smithy.model.shapes.CollectionShape
-import software.amazon.smithy.model.shapes.ListShape
-import software.amazon.smithy.model.shapes.MapShape
-import software.amazon.smithy.model.shapes.MemberShape
-import software.amazon.smithy.model.shapes.SetShape
-import software.amazon.smithy.model.shapes.Shape
-import software.amazon.smithy.model.shapes.TimestampShape
-import software.amazon.smithy.model.traits.SparseTrait
-import software.amazon.smithy.model.traits.TimestampFormatTrait
-import software.amazon.smithy.swift.codegen.ClientRuntimeTypes
-import software.amazon.smithy.swift.codegen.SwiftTypes
-import software.amazon.smithy.swift.codegen.SwiftWriter
-import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
-import software.amazon.smithy.swift.codegen.integration.serde.MemberShapeDecodeGeneratable
-import software.amazon.smithy.swift.codegen.integration.serde.TimestampDecodeGenerator
-import software.amazon.smithy.swift.codegen.integration.serde.TimestampHelpers
-import software.amazon.smithy.swift.codegen.model.defaultValue
-import software.amazon.smithy.swift.codegen.model.hasTrait
-import software.amazon.smithy.swift.codegen.model.isBoxed
-import software.amazon.smithy.swift.codegen.model.toMemberNames
-import software.amazon.smithy.swift.codegen.removeSurroundingBackticks
-
-/*
-Includes functions to help render conformance to Decodable protocol for shapes
- */
-abstract class MemberShapeDecodeGenerator(
```

Comment:
This JSON implementation is deleted and the unified member shape decode codegen is used instead.